autossh/config.yaml

```
@@ -21,15 +21,20 @@ options:
   hostname: ''
   ssh_port: 22
   username: autossh
-  remote_forwarding:
-    - '127.0.0.1:8123:172.30.32.1:8123'
+  forward_local_ip_address: "172.17.0.1"
```

Shouldn't that be this default?

```suggestion
  forward_local_ip_address: "172.30.232.1"
```